src/plugins/preauth/pkinit/pkinit_srv.c

```diff
@@ -51,6 +79,47 @@ pkinit_find_realm_context(krb5_context context,
                           krb5_kdcpreauth_moddata moddata,
                           krb5_principal princ);
 
+static void
+free_ai_list(char **ai_total)
```

Comment:
"ai_total" seems like a relic from when this was part of another piece of code.  "list" would be more natural.